mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -55,22 +43,16 @@ cfg_native!(
                   TxFilter};
 );
 
-#[cfg(not(target_arch = "wasm32"))]
-pub type OnCompactBlockFn<'a> = dyn FnMut(TonicCompactBlock) -> Result<(), MmError<UpdateBlocksCacheErr>> + Send + 'a;
-
-#[cfg(target_arch = "wasm32")]
-#[allow(unused)]
-pub type OnCompactBlockFn<'a> = dyn FnMut(String) -> Result<(), MmError<UpdateBlocksCacheErr>> + Send + 'a;
-
 #[async_trait]
 pub trait ZRpcOps {
```

Comment:
left note about doc coms in this [PR](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1922), I believe it will be merged first. Just a note.

Reply:
yep, seen thanks @laruh